src/sql/ConfigEditor/ConfigSelect.test.tsx

```diff
@@ -28,19 +29,17 @@ describe('SQLTextInput', () => {
   });
 
   it('should show disabled select if passed disabled as prop', () => {
-    const label = 'foo-id';
-    render(<ConfigSelect {...props} disabled={true} label={label} />);
-    const selectEl = screen.getByLabelText(label);
+    render(<ConfigSelect {...props} disabled={true} aria-label={props['aria-label']} />);
```

Comment:
does the aria-label need to be set separately here or is it set by the `props` field already?